### PR TITLE
Changes:

### PR DESCRIPTION
**Pull request by KalpeshThakare**
Changes:
* TableView Row Swipe Action Configuration
* created the extension for UISwipeActionsConfiguration

### CODE REVIEW

File: Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlexHmelevskiAG) [inline Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist]: why do we add this?

File: Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```
Comment (AlexHmelevskiAG) [inline Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist]: same here

File: Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist (hunk 1 above)
Comment (KalpeshThakare) [reply Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist]: > why do we add this?

deleted

File: Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist (hunk 2 above)
Comment (KalpeshThakare) [reply Example/EZSource.xcodeproj/xcuserdata/kalpeshthakare.xcuserdatad/xcschemes/xcschememanagement.plist]: deleted

**Pull request merged**